googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/googleprediction/GooglePredictionConnectorIntegrationTest.java

```diff
@@ -483,9 +482,7 @@ public void testDeleteTrainedModelWithNegativeCase() throws Exception {
                         apiRequestHeadersMap);
         
         Assert.assertEquals(apiRestResponse.getHttpStatusCode(), esbRestResponse.getHttpStatusCode());
-        Assert.assertEquals(esbRestResponse.getBody().getJSONObject("error").getString("message"), apiRestResponse
-                .getBody().getJSONObject("error").getString("message"));
-        
+
```

Comment:
Fixed in https://github.com/wso2-extensions/esb-connector-googleprediction/blob/master/src/test/java/org/wso2/carbon/connector/integration/test/googleprediction/GooglePredictionConnectorIntegrationTest.java